chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1384,6 +1385,14 @@ impl Get<u64> for TimestampProvider {
     }
 }
 
+parameter_types! {
+    // Chain ID of Holesky.
+    pub RewardsEthereumSovereignAccount: AccountId =
+        tp_bridge::EthereumLocationsConverterFor::<AccountId>::convert_location(
+            &Location::new(1, [GlobalConsensus(Ethereum { chain_id: 17_000 } )])
```

Comment:
I believe we already have a chain_id somewhere in `chains/orchestrator-relays/runtime/dancelight/constants/src/lib.rs`

Can we use that?